x-pack/plugins/security_solution/public/exceptions/components/exceptions_list_card/index.tsx

```diff
@@ -131,7 +137,12 @@ export const ExceptionsListCard = memo<ExceptionsListCardProps>(
                       >
                         <EuiFlexItem grow>
                           <EuiText size="m">
-                            <EuiLink data-test-subj="exception-list-name">{listName}</EuiLink>
+                            <EuiLink
+                              data-test-subj="exception-list-name"
+                              onClick={goToExceptionDetail}
+                            >
+                              {listName}
+                            </EuiLink>{' '}
```

Comment:
`{' '}` I think this space is unwanted :D

Reply:
ah interesting - not sure how that got in there. Thanks!